packages/plugins/test/mocha/simulate.spec.ts

```diff
@@ -1,118 +1,132 @@
-// import chai, { expect } from 'chai'
```

Comment:
This file is unchanged except there's some code that skips running these tests if on a feature branch in CI